inabox/tests/setup_disperser_harness.go

```diff
@@ -207,19 +214,34 @@ func SetupDisperserHarness(
 		logger.Warn("Relay count is not specified, skipping relay setup")
 	}
 
+	// Start encoder v2 goroutine
+	encoderAddress, err := startEncoderV2(ctx, harness, config)
+	if err != nil {
+		return nil, fmt.Errorf("failed to start encoder v2: %w", err)
+	}
+
 	// Start controller goroutine
-	if err := startController(ctx, ethClient, config.OperatorStateSubgraphURL, harness, config); err != nil {
+	if err := startController(
+		ctx,
+		ethClient,
+		config.OperatorStateSubgraphURL,
+		encoderAddress,
+		harness,
+		config,
+	); err != nil {
```

Comment:
still think all of these should be refactored to return the created structs which we can then manually add to the harness. Prefer when inplace modifications are explicit. But can all be done in a future PR.